client/src/Clusters/TopCharts/SingleAppContainer.test.js

```diff
@@ -0,0 +1,18 @@
+import SingleAppContainer from './SingleAppContainer.js';
+
+
+const renderSingleAppContainer = () =>
+    render(
+        <SingleAppContainer 
+            listNum= {1}
```

Comment:
remove the spaces after the =

The convention in react is to do

```
listNum={1}
```

Same below